cmd/buildah/bud.go

```diff
@@ -26,8 +26,8 @@ func init() {
 	budDescription := `
   Builds an OCI image using instructions in one or more Dockerfiles.
 
-  If no arguments specified, it will assume the current working directory as
-  build context, which should contain the Dockerfile.`
+  If no are arguments specified, it will assume the current working directory
```

Comment:
"are" should be moved behind "arguments"

Reply:
Fixed with the comment below.